Form/Type/SimpleMoneyType.php

```diff
@@ -31,7 +31,7 @@ public function __construct(PairManagerInterface $pairManager, $decimals)
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('tbbc_amount', new TextType())
+            ->add('tbbc_amount', 'Symfony\Component\Form\Extension\Core\Type\TextType')
```

Comment:
isn't it written more shortly with `TextType::class` ? (same for other places where a whole string of namespace is put )